packages/backend/src/config/app.ts

```diff
@@ -73,7 +73,10 @@ export const Config = {
   webhookSecret: process.env.WEBHOOK_SECRET, // optional
   webhookTimeout: envInt('WEBHOOK_TIMEOUT', 2000), // milliseconds
 
-  webMonetizationWithdrawalCron: process.env.WEB_MONETIZATION_WITHDRAWAL_CRON, // optional
+  withdrawalThrottleDelay:
```

Comment:
Why not use `envInt`?

Reply:
I wanted `withdrawalThrottleDelay` to be optional without a default (`number | undefined`) and didn't want to change `envInt`'s return type